### PR TITLE
Allow opening in-app notifications in external browser

### PR DESCRIPTION
**Pull request by Pedro-vk**
### Description

_Allow opening in-app notifications in external browser_

### Tested

_Tested on Android_

### How others should test

_Creating or editing a notification in Firebase_

### Related issues

- Fixes https://github.com/celo-org/celo-monorepo/issues/7929

### Backwards compatibility

_It is backward compatible, if the notification doesn't specify the external parameter, it takes "false" as default, that behaves like previous versions_

### CODE REVIEW

**Review by etuleu — APPROVED** [review]

File: packages/mobile/src/home/reducers.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ export interface Notification {
   maxVersion?: string
   countries?: string[]
   blockedCountries?: string[]
+  external?: boolean
```
Comment (jeanregisser) [inline packages/mobile/src/home/reducers.ts:21]: Let's rename to keep it consistent with the naming used for push notification: https://github.com/celo-org/wallet/blob/c2cf2292a01d11fe53f23dae91f9a463b46da747/packages/mobile/src/firebase/notifications.ts#L96
```suggestion
  openExternal?: boolean
```

**Review by jeanregisser — CHANGES_REQUESTED** [review]

**Review by jeanregisser — COMMENTED** [review]
Thanks, could you also update `NotificationBox.test.tsxt` to cover this new use case?

**Comment by Pedro-vk** [comment]
> Thanks, could you also update `NotificationBox.test.tsxt` to cover this new use case?

Done!

**Review by jeanregisser — APPROVED** [review]
Shiiip it 👍 🐑 :shipit:

**Comment by Celoqa** [comment]
Hey @MuckT  I have verified this issue using  Android internal build V1.16.1 & Test Flight build V1.16.1(67) & updated our observation in below sheet.
**Sheet Link:** https://docs.google.com/spreadsheets/d/1wYi5kbe_NUXuhc9xtcbT9dtyrVbPXqNBrAETfSx0R58/edit#gid=1119817571&range=O2

**New bug:**

- https://github.com/celo-org/wallet/issues/859

**Verified On Devices:** Redmi Note 8 (9.0),Redmi Note 8 (10.0), Vivo V11 pro (10)
CC: @Pedro-vk

**Pull request merged**